website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -65,39 +73,37 @@ metrics:
       - field: is_paying
         value: true
 
-    meta: {}
+    meta: {team: Finance}
```

Comment:
I feel like the examples in the properties table should match this example YAML file.